core/src/main/java/org/apache/accumulo/core/client/admin/NewTableConfiguration.java

```diff
@@ -121,8 +128,22 @@ public NewTableConfiguration setProperties(Map<String,String> prop) {
    */
   public NewTableConfiguration enableSampling(SamplerConfiguration samplerConfiguration) {
     requireNonNull(samplerConfiguration);
-    SamplerConfigurationImpl.checkDisjoint(properties, samplerConfiguration);
-    this.samplerConfiguration = samplerConfiguration;
+    Map<String,String> tmp = new SamplerConfigurationImpl(samplerConfiguration).toTablePropertiesMap();
+    checkDisjoint(properties, tmp, summarizerProps);
+    this.samplerProps = tmp;
```

Comment:
Seems odd to be checking disjoint with `summarizerProps` but then updating `samplerProps` instead. Maybe javadoc on those member variables would help describe what they each hold.